pontoon/contributors/static/js/status.js

```diff
@@ -0,0 +1,17 @@
+$(function () {
+  $('#account-status-form').on('submit', function (event) {
+    event.preventDefault();
+
+    const form = $(this);
+    $.ajax({
+      url: form.attr('action'),
+      type: 'POST',
+      data: {
+        csrfmiddlewaretoken: $('body').data('csrf'),
+      },
+      success: function () {
+        location.reload();
+      },
+    });
+  });
+});
```

Comment:
I'd move this code to the `profile.js` file, where we keep all the JS specific for the Profile page.